### PR TITLE
Fix integration workflow

### PR DESCRIPTION
**Pull request by niklassiemer**

### CODE REVIEW

**Review by samwaseda — APPROVED** [review]
My manual test also passed (i.e. integration tests started running when I added `integration`). Good to go by me.

**Review by samwaseda — CHANGES_REQUESTED** [review]
Sorry I just realized that the notebooks are not running.

File: .github/workflows/notebooks.yml [hunk 1]
```diff
@@ -18,4 +19,4 @@ jobs:
         with:
           python-version: '3.11'
           env-files: .ci_support/environment-notebooks-integration.yml
-          exclusion-file: .ci_support/exclude
\ No newline at end of file
+          exclusion-file: .ci_support/exclude
```
Comment (samwaseda) [inline .github/workflows/notebooks.yml:22]: I didn't touch this line... But I guess it only added EOF so I don't think it disturbs

**Review by samwaseda — APPROVED** [review]

**Comment by mbruns91** [comment]
It seems like this this is working now, right?

**Pull request merged**